ptvsd/wrapper.py

```diff
@@ -1012,6 +1012,31 @@ def on_evaluate(self, request, args):
         safe_repr_provider.convert_to_hex = False
         self.send_response(request, **response)
 
+    @async_handler
+    def on_setExpression(self, request, args):
+        # TODO: docstring
+
+        vsc_fid = int(args['frameId'])
+        pyd_tid, pyd_fid = self.frame_map.to_pydevd(vsc_fid)
+
+        lhs_expr = args.get('expression')
+        rhs_expr = args.get('value')
+        expr = '%s = %s' % (lhs_expr, rhs_expr)
```

Comment:
I think this should be:
```py
%s = (%s)
```
To prevent accidental or creative abuses, e.g. if someone stuffs a semicolon into `rhs_expr`.